oss_src/sframe/sframe_rows.cpp

```diff
@@ -32,16 +34,36 @@ void sframe_rows::clear() {
 
 void sframe_rows::save(oarchive& oarc) const {
   oarc << m_decoded_columns.size();
-  for (auto& i : m_decoded_columns) oarc << *i;
+  oarchive temp_inmemory_arc;
+  for (auto& i : m_decoded_columns) {
+    v2_block_impl::block_info info;
```

Comment:
This is just for sframe_rows. sframe_rows is normally an internal object used to emulate vector<rows> when the actual representation is vector<columns>. But on occasion it is very useful to be able to serialize and deserialize these things.